code/modules/mob/mob_defines.dm

```diff
@@ -76,14 +76,17 @@
 	var/m_intent = "run"//Living
 	var/lastKnownIP = null
 	var/atom/movable/buckled = null//Living
+	var/obj/item/l_hand = null//Living
+	var/obj/item/r_hand = null//Living
+	var/obj/item/weapon/back = null//Human/Monkey
```

Comment:
I'm not sure what you mean by override `get_equipped_items()` for `/carbon` and `/human`, what file should I change? What changes should I make to override this proc?